src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringInterpolation.cs

```diff
@@ -18,7 +18,10 @@ private BoundExpression RewriteInterpolatedStringConversion(BoundConversion conv
             Debug.Assert(conversion.ConversionKind == ConversionKind.InterpolatedString);
             BoundExpression format;
             ArrayBuilder<BoundExpression> expressions;
-            MakeInterpolatedStringFormat((BoundInterpolatedString)conversion.Operand, out format, out expressions);
+            // TODO: Last argument could be true for some cases
+            // e.g. - ((FormattableString)$"Now: {DateTime.Now}").ToString(CultureInfo.GetCulture("some-lang"))
+            //      - FormattableString.Invariant($"Now: {DateTime.Now}")
+            MakeInterpolatedStringFormat((BoundInterpolatedString)conversion.Operand, out format, out expressions, false);
```

Comment:
Please use the parameter name for `false`.

Reply:
It is better to just remove it (because it is an optional parameter)?